README.md

```diff
@@ -155,6 +158,7 @@ One could also take advantage of the cache effect. If the data is not saturating
 | Instrumentation  |    72 ns |
 | AsyncFileHandler |  1100 ns |
 | FileHandler      | 45000 ns |
+| Beans Enabled AsyncFileHandler |  1400 ns |
```

Comment:
Should this be Beans-Enabled?
Could align the column size for other rows?

Reply:
asyncfilehandler adds latencies too.